ts/features/itwallet/presentation/remote/utils/itwRemoteTypeUtils.ts

```diff
@@ -0,0 +1,29 @@
+import { Credential } from "@pagopa/io-react-native-wallet";
+import { ClaimDisplayFormat } from "../../../common/utils/itwClaimsUtils";
+
+/**
+ * Type for the parameters requested to start the presentation flow
+ */
+export type ItwRemoteRequestPayload =
```

Comment:
This change breaks deep linking because the parameters case was updated from `snake_case` to `camelCase`.
The QR Code and deep link payloads need to remain consistent to ensure compatibility and avoid unnecessary manipulation.
Was there a specific reason for changing the case?

Reply:
The case was changed mainly for consistency with variable names across the code, to avoid mixing snake case and camel case.
I do not think this breaks existing deep links, because we extract the parameters with `params.get("param_name")` and I kept the snake case there.
I do agree on the extra manipulation required, so I'm ok with restoring the snake case.

Reply:
Ok I see the issue. Now we don't need the custom `io-ts` type because the payload validation happens in `startFlowFromQR` from `io-react-native-wallet`, so that function needs to return the snake case validated payload to avoid mapping between snake case and camel case.